Sources/PenguinParallel/NonblockingThreadPool/NonBlockingThreadPool.swift

```diff
@@ -467,9 +467,9 @@ fileprivate final class PerThreadState<Environment: ConcurrencyPlatform> {
   var isCancelled: Bool { pool.cancelled }
 
   func steal() -> Task? {
-    let r = Int(rng.next())
-    var selectedThreadId = fastFit(r, into: pool.totalThreadCount)
-    let step = pool.coprimes[fastFit(r, into: pool.coprimes.count)]
+    let r = rng.next() as UInt32
```

Comment:
Do you still need the `as UInt32`?

Reply:
Nope, one more fix and then this should be ready to go.